tests/functional/api/BackbeatServer.js

```diff
@@ -341,27 +357,27 @@ describe('Backbeat Server', () => {
             ['a'],
             [{
                 // Missing bucket property.
```

Comment:
Comments to be updated to reflect new property names (capitalized, StorageClass)